tests/models/test_onnx.py

```diff
@@ -14,10 +14,16 @@
 import os
 
 import numpy as np
-import onnxruntime
 import pytest
 import torch
 
+try:
+    import onnxruntime
```

Comment:
Why we require it for testing... it does not make sense to ignore this testing...

Reply:
I rain into install issues on mac if I recall correctly, @tchaton is this what you're trying to address here?

Reply:
I rain into install issues on mac if I recall correctly, @tchaton is this what you're trying to address here?

Reply:
what issue, can you share details?

Reply:
```
(.venv) ➜  pytorch-lightning git:(bugfix/dict_wo_loss_error) ✗ python
Python 3.8.3 (default, Oct 14 2020, 10:33:18) 
[Clang 12.0.0 (clang-1200.0.32.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import onnxruntime
/Users/thomas/Documents/projects/pytorch-lightning/.venv/lib/python3.8/site-packages/onnxruntime/capi/_pybind_state.py:14: UserWarning: Cannot load onnxruntime.capi. Error: 'dlopen(/Users/thomas/Documents/projects/pytorch-lightning/.venv/lib/python3.8/site-packages/onnxruntime/capi/onnxruntime_pybind11_state.so, 2): Library not loaded: /usr/local/opt/libomp/lib/libomp.dylib
  Referenced from: /Users/thomas/Documents/projects/pytorch-lightning/.venv/lib/python3.8/site-packages/onnxruntime/capi/onnxruntime_pybind11_state.so
  Reason: image not found'.
  warnings.warn("Cannot load onnxruntime.capi. Error: '{0}'.".format(str(e)))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/thomas/Documents/projects/pytorch-lightning/.venv/lib/python3.8/site-packages/onnxruntime/__init__.py", line 13, in <module>
    from onnxruntime.capi._pybind_state import get_all_providers, get_available_providers, get_device, set_seed, \
ImportError: cannot import name 'get_all_providers' from 'onnxruntime.capi._pybind_state' (/Users/thomas/Documents/projects/pytorch-lightning/.venv/lib/python3.8/site-packages/onnxruntime/capi/_pybind_state.py)
```

Reply:
Hey @Borda, here is the bug I have locally. It is preventing me to `Discover Tests`.

Reply:
but it is happening only for you locally, or have you seen this also on CI?